src/NorthwindWebsite/Core/Utils/StringUtils.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.WebUtilities;
+using System.Text;
+
+namespace NorthwindWebsite.Core.Utils
+{
+    public static class StringUtils
+    {
+        public static string EncodeCode(string code) => 
+            WebEncoders.Base64UrlEncode(Encoding.UTF8.GetBytes(code));
```

Comment:
1. Probably `Code` suffix in method is not really suitable, it's better to rename to `EncodeToWeb`
2. Since you utility is responsible for `Web` encoding, it's better to move it to _Presentation_ layer and rename class to `WebUtils`